charts/datastore/templates/persistentVolume.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if (eq .Values.datastore.inMemoryDb false) }}
```

Comment:
What if inMemoryDb is not defined? This condition means it will make an inMemoryDb.
I suggested using `eq` if you wanna make sure it's exactly false, but it needs to make sense to do that in the context of what you're trying to do

Reply:
it always defined in the `values.yaml`

Reply:
What if they don't use the values.yaml? We do this regularly when we don't use it as a chart but as yamls...

Reply:
but here we are talking on a chart. 

In yaml you can do whatever you want and i have no way to save you with render-time checks opposed to chart.

Reply:
I don't understand, why not use `not`?? It would cover the case of having it not defined and behave correctly...

Reply:
used default instead